misk-crypto/README.md

```diff
@@ -34,6 +34,7 @@ tinkey create-keyset --key-template AES256-GCM --master-key-uri aws-kms://arn:km
 Then, to specify a new `Cipher` key called "myKey", add the following in your app's configuration file:
 ```$yaml
 crypto:
+  kms_uri: "aws-kms://arn:kms:<region>:<account-id>:key/<key-id>"
```

Comment:
nit: `key_uri` instead? kms is specifically Amazon KMS isn't it?

Reply:
KMS is acronym for Key Management System, both Google and Amazon use it.